chart/templates/controller-manager.yaml

```diff
@@ -202,6 +202,7 @@ spec:
         - --leader-elect
         - --wds-name={{.Values.ControlPlaneName}}
         - --api-groups={{.Values.APIGroups}}
+        - -v=2
```

Comment:
Maybe also allow the verbose level to come from values?  Will be convenient for debugging

Reply:
I suggest we leave this to another PR as there are changes introduced by #1779 I think we should merge first.